recipes/beauty/all/conanfile.py

```diff
@@ -0,0 +1,82 @@
+import os
+from conans import ConanFile, tools
+from conan.tools.cmake import CMake
```

Comment:
```suggestion
from conan.tools.cmake import CMake
from conan.tools.microsoft import is_msvc
```
https://docs.conan.io/en/latest/reference/conanfile/tools/microsoft.html#conan-tools-microsoft-is-msvc

Reply:
Done